cmd/vic-machine/common/utils.go

```diff
@@ -31,3 +34,27 @@ func LogErrorIfAny(clic *cli.Context, err error) error {
 	log.Errorf("%s %s failed: %s\n", clic.App.Name, clic.Command.Name, errors.ErrorStack(err))
 	return cli.NewExitError("", 1)
 }
+
+// CheckUnsupportedChars returns an error if string contains special characters
+// https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=2046088
+func CheckUnsupportedChars(s string) error {
+	re := regexp.MustCompile(`%|&|\*|\$|#|@|!|\\|/|:|\?|"|<|>|;|'|\|`)
+	return checkUnsupportedChars(s, re)
+}
+
+// CheckUnsupportedCharsDatastore returns an error if a path string contains special characters
+// Same as CheckUnsupportedChars but allows / and :
+func CheckUnsupportedCharsDatastore(s string) error {
+	re := regexp.MustCompile(`%|&|\*|\$|#|@|!|\\|\?|"|<|>|;|'|\|`)
```

Comment:
Please define constants for the two regex, and add comment for their difference, to make it easy to understand and update